core/steam_client.gd

```diff
@@ -648,7 +739,12 @@ func _install_update(app_id: String, path: String = "") -> bool:
 		logger.info("Install progress: " + str(output))
 		for line in output:
 			if line.contains("Success! "):
-				success.append(true)
+				success.append("installed")
+				continue
+			if line.contains("not online or not logged"):
+				logger.info(line)
+				if not self.logged_in_user.is_empty():
```

Comment:
This logic just stands out intuitivly as backwards, can you explain in a comment?